AK/NumberFormat.cpp

```diff
@@ -110,25 +111,28 @@ DeprecatedString human_readable_digital_time(i64 time_in_seconds)
     return builder.to_deprecated_string();
 }
 
-DeprecatedString human_readable_integer(i64 number)
+DeprecatedString human_readable_integer(const i64 number)
```

Comment:
Please use west `const`

Reply:
east :)

Reply:
but in this case there's literally no point in making the parameter const.